.CI/create_feedstocks

```diff
@@ -10,7 +10,7 @@ if [ -n "$GH_TOKEN" ]; then
     conda config --add channels conda-forge
     conda install --yes --quiet conda-smithy
     # Temporary workaround b/c conda-build-all is not ready for the latest conda-build.
-    conda install -n root --yes --quiet conda-build=1.20.1
+    conda install -n root --yes --quiet conda-build=1.20.3
```

Comment:
Try removing this line (and the comment above). The temporary workaround should no longer be necessary.